### DISCUSSION TITLE
Cut stream Chapters into individual videos

### DISCUSSION THREAD

**Discussion by furensic** [MrBrax/LiveStreamDVR#408] [Q&A]
Hey guys, lets say im recording a streamer that stream 2 hours League of Legends and 5 Hours Just chatting, how can i setup that when i set a chapter to end at 2 hours, it creates a seperate video from that? Is that even possible or not yet a function

**Comment by MrBrax** [comment]
there's nothing built in since chapter times are usually off, but you can use lossless-cut with the file generated to help you on the way
